staging/src/k8s.io/library-go/sig-scheduling/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I thought we wanted an import path more like `k8s.io/library-go/scheduling/...` (drop the `sig-` bit)